maven-release-plugin/src/main/java/org/apache/maven/plugins/release/PerformReleaseMojo.java

```diff
@@ -179,16 +139,6 @@ public void execute()
                 releaseDescriptor.setScmSourceUrl( connectionUrl );
             }
 
-            if ( username != null )
-            {
-                releaseDescriptor.setScmUsername( username );
-            }
-
-            if ( password != null )
-            {
-                releaseDescriptor.setScmPassword( password );
-            }
-
```

Comment:
Sure this should be removed?
Now the releaseDescriptor no longer has these values.
There is _no_ scenario when during the perform release the username/password are needed?
If there is no such scenario: Why do you keep the scm source url? I the ScmSourceUrl is needed then I would suspect the username and password are also needed in some cases.

Reply:
This is IMHO already part of `AbstractScmReleaseMojo.createReleaseDescriptor()`

Reply:
@kwin You are right.